utils/jest/style.stub.js

```diff
@@ -1,3 +1,3 @@
-// TODO: use https://github.com/keyanzhang/identity-obj-proxy
+// see: https://facebook.github.io/jest/docs/tutorial-webpack.html
 
-module.exports = {};
+module.exports = 'identity-obj-proxy';
```

Comment:
I don't think this would work, right? It would just export a string literal. You'd have to either import and re-export the module here (messy) or change the mapping in moduleNameMapper in jest.js

How did you test this?